planner/core/common_plans.go

```diff
@@ -297,6 +298,7 @@ func (e *Execute) getPhysicalPlan(ctx context.Context, sctx sessionctx.Context,
 		} else {
 			planCacheCounter.Inc()
 		}
+		stmtCtx.AddPlanCacheHitInfo(true)
```

Comment:
How about setting this value in `setFoundInPlanCache()`?

Reply:
@zz-jason  Hard to decide. Merging two functions does make code tidier. However, if we want to add more parameters about `plan cache` to `information_schema`, more variables may be added to the function. Then the definition of `setFoundInPlanCache` may be too vague, compared with other environment variable set functions.

Reply:
Why can't we just use `FoundInPlanCache`?